app/styles/app.scss

```diff
@@ -82,12 +106,23 @@ body {
   width: 100%;
 }
 
+#footer {
+  padding-top: 50px;
+  a{
+    color:black;
+  }
+  center{
+    color:white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-color, color<br>Colon after property should be followed by one space